### ISSUE TITLE
Why the highest training accuracy happened in Iteration 0 ? 

### ISSUE THREAD

**Issue by xiaoxiongli** [daijifeng001/R-FCN#40]
Now I am training the 4th R-FCN stage (totally 4 stage, with OHEM, 229 category LOGOs), and I met a situation that the highest training accuracy(0.949) happened in Iteration 0 (that means just use the stage 2' model).  And after training several iteration(max 110000 as you set), the final accuray is still lower than Iteration 0. 
         I feel confuse that Why this happens..., if this always happens, we even do not need the last training stage...,  this problem bother me a lot......, help..
         I try 3 kind of base learning rate: 0.01, 0.001, 0.0001.  the situation I mentioned just now is almost same. just like this:
        
------------------------- Iteration 0 -------------------------
Training : **# accuracy 0.949**, loss (cls 0.124, reg 0.243)
Testing  : accuracy 0.926, loss (cls 0.201, reg 0.223)

------------------------- Iteration 2000 -------------------------
Training : accuracy 0.938, loss (cls 0.148, reg 0.0596)
Testing  : accuracy 0.927, loss (cls 0.2, reg 0.0525)

------------------------- Iteration 4000 -------------------------
Training : accuracy 0.938, loss (cls 0.147, reg 0.0418)
Testing  : accuracy 0.927, loss (cls 0.199, reg 0.0504)

------------------------- Iteration 6000 -------------------------
Training : accuracy 0.94, loss (cls 0.144, reg 0.0399)
Testing  : accuracy 0.927, loss (cls 0.198, reg 0.0497)

.....
.....

------------------------- Iteration 98000 -------------------------
Training : accuracy 0.943, loss (cls 0.135, reg 0.0315)
Testing  : accuracy 0.929, loss (cls 0.193, reg 0.0431)

------------------------- Iteration 100000 -------------------------
Training : accuracy 0.943, loss (cls 0.137, reg 0.0317)
Testing  : accuracy 0.929, loss (cls 0.193, reg 0.0431)

------------------------- Iteration 102000 -------------------------
Training : **# accuracy 0.943**, loss (cls 0.134, reg 0.0317)
Testing  : accuracy 0.929, loss (cls 0.193, reg 0.0429)

....
....

**Comment by northeastsquare** [comment]
What about your test accuracy?

**Comment by ouxinyu** [comment]
@xiaoxiongli I see you use " 4th R-FCN stage" to train the R-FCN, and how to train the RPN with Resnet101 you are. Can you share the codes? thank you!

**Comment by xiaoxiongli** [comment]
@ouxinyu I recommend you that use the R-FCN's python version, it is trained in a end2end method(not 4-stage training), the end2end training method is 1.5x faster than 4-stage training. and it is more simpler: only write the network down as a single model and just train it. So i am focus on it now and i am learning Python now. I got a mAP=97.2(faster-rcnn is about mAP=90) in my own training data and test data(229 category logo, about 3w images) with our 4-stage(alternating optimization) training. R-FCN is really good. For the code commit to github or release to anyone else, I need some discuss with my CEO, So Please try the RFCN's Python version first. bless.

**Comment by xiaoxiongli** [comment]
@northeastsquare for my own training data(229 category logo, about 3w images) , compare to faster-rcnn, the mAP change from 90 to 97.

**Comment by psy770** [comment]
@xiaoxiongli can you give me some advice for fine-tuning R-FCN ?

**Comment by msukoz** [comment]
hi, @xiaoxiongli, how was your training using R-FCN on your own data via 4 step training? I now want to use my own data to finetune on R-FCN via 4 step training, but it crashes in voc0712_trainval_sp.m. Author gives 6 .mat file in folder "self_proposal_data", but my own data can not use these .mat , right? So can I get my own .mat(self_proposal_data). Thank you.

**Comment by Chen94yue** [comment]
你好，我碰到了相同的问题。一旦不使用stage5的输出接ROI，accuracy就出现异常，请问你是怎么解决的啊。
